minefield_game/player_header.h

```diff
@@ -5,9 +5,11 @@ struct Mine;
 struct Player
 {
     std::vector<Mine> mines;
-    std::vector<Cell> guesses; //recordar que la cantidad de adivinanzas que tenga depende la cantidad de minas no descubiertas del contrincante
-    int remainingMines; //la cantidad de minas que puedo usar o colocar aún
+    std::vector<Cell> guesses;
+    std::vector<Cell> disabledMineSpots; // locations where own mines were destroyed
+    int remainingMines; // the number of mines the player can still place or use
 };
+
 struct Mine
 {
     Cell cell;

```

Comment:
#could rename this field such that this strong type makes more sense. i.e. `Cell location`, then you'll refer to it as `mine.location`, which looks nice!